### PR TITLE
Imroved Code integration.

### PR DESCRIPTION
**Pull request by simonmika**
Two minor improvements on VS Code integration.

### CODE REVIEW

File: test.sh [hunk 1]
```diff
@@ -36,7 +36,7 @@ do
 done
 echo "Main: ./test/Tests.ooc" >> "$TESTS_USE_FILE"
 rm -f .libs/tests-linux64.*
-rock --allerrors -q -lpthread --gc=off $ARGS $FLAGS $TESTS_USE_FILE && ./Tests
+rock -q -lpthread --gc=off $ARGS $FLAGS $TESTS_USE_FILE && ./Tests
```
Comment (davidhesselbom) [inline test.sh]: What if I'm not using Code?

Comment (simonmika) [inline test.sh]: That is actually why I went for not displaying all errors. It is not that help full then.

**Comment by simonmika** [comment]
There is trouble with the `ThreadPool`, nothing with this PR.

File: test.sh (hunk 1 above)
Comment (simonmika) [inline test.sh]: You can always call `./test.sh all --allerrors`.

**Pull request merged**